orbit/pkg/table/extension_windows.go

```diff
@@ -3,20 +3,54 @@
 package table
 
 import (
+	"fmt"
+
 	cisaudit "github.com/fleetdm/fleet/v4/orbit/pkg/table/cis_audit"
 	mdmbridge "github.com/fleetdm/fleet/v4/orbit/pkg/table/mdm"
 	"github.com/fleetdm/fleet/v4/orbit/pkg/table/windowsupdatetable"
+	"golang.org/x/sys/windows/registry"
 
 	"github.com/osquery/osquery-go"
 	"github.com/osquery/osquery-go/plugin/table"
 )
 
-func PlatformTables(_ PluginOpts) []osquery.OsqueryPlugin {
-	return []osquery.OsqueryPlugin{
+func PlatformTables(_ PluginOpts) ([]osquery.OsqueryPlugin, error) {
+	plugins := []osquery.OsqueryPlugin{
 		// Fleet tables
-		table.NewPlugin("mdm_bridge", mdmbridge.Columns(), mdmbridge.Generate),
 		table.NewPlugin("cis_audit", cisaudit.Columns(), cisaudit.Generate),
 
 		windowsupdatetable.TablePlugin(windowsupdatetable.UpdatesTable, osqueryLogger), // table name is "windows_updates"
 	}
+
+	windowsServer, err := IsWindowsServer()
+	if err != nil {
+		return nil, err
+	}
+
+	if !windowsServer {
+		plugins = append(plugins, table.NewPlugin("mdm_bridge", mdmbridge.Columns(), mdmbridge.Generate))
+	}
+
+	return plugins, nil
+}
+
+func IsWindowsServer() (bool, error) {
+	// If the registry can't be read, it's safer to assume we're a
+	// server and not load the broken table
```

Comment:
Out of curiosity: Isn't `IsWindowsServer` returning an error (and `false`) if it fails to read the registry?

Reply:
Yes

Reply:
So the comment doesn't match the implementation? A nit, just trying to understand the intent.

Reply:
Oh I didn't even notice the comment, that was from a previous iteration where it would return only a bool. I will create a PR to remove that comment, sorry about the confusion and nice catch 🙂